.circleci/config.yml

```diff
@@ -13,22 +13,29 @@ commands:
       - run: brew install protobuf cmake ccache libtool boost-python libspatialite pkg-config luajit curl wget czmq lz4 spatialite-tools unzip
 
 jobs:
-  build-base:
+  build-docker-images:
     docker:
       - image: docker:18.06.3-ce-git
     steps:
       - setup_remote_docker
       - checkout
       - run:
-          name: Build Valhalla base Dockerfile
+          name: Build Valhalla Docker Images
           command: |
             docker build -f ./docker/Dockerfile-build --tag valhalla/valhalla:build-latest ./docker
+            if [[ "${CIRCLE_BRANCH}" == "master" ]] || [[ ! -z "${CIRCLE_TAG}" ]]; then
+              docker build -f ./docker/Dockerfile-run --tag valhalla/valhalla:run-latest .
+            fi
       - run:
-          name: Push Valhalla base image
+          name: Push Valhalla Docker Images
           command: |
+            echo "$DOCKERHUB_PASS" | docker login -u "$DOCKERHUB_USERNAME" --password-stdin
             if [ "${CIRCLE_BRANCH}" == "master" ]; then
-              echo "$DOCKERHUB_PASS" | docker login -u "$DOCKERHUB_USERNAME" --password-stdin
               docker push valhalla/valhalla:build-latest
+              docker push valhalla/valhalla:run-latest
+            elif [[ ! -z "${CIRCLE_TAG}" ]]; then
```

Comment:
i figured tagging was a good thing to do too. we do this rarely but when it fails we'll get an email and can fix it manually